docs/source/example_notebooks/dowhy_causal_prediction_demo.ipynb

```diff
@@ -0,0 +1,1267 @@
Ex3WTlCTEtBOTBnNnNtSU1uWUwlMkJJa2xKcWxicFR5bktwUGRvVjhFWDd3VTByS09PYmZpSG1MSnpsaklqZkElMkJQNFJjUXhVdnlhOVhWdUIxcm9mTFRXRVFCZDlMRkZLTkhuVjhKc3ElMkIzQ1pFRjdxN1VrQUJRQ3lFUGdUaVY0JTJCY1pWRTk1TGpFNmg0R0xCaGFnR09jJTJGUGYlMkI1dCUyQmpyeDJQaXdDU3E0YXclMkJjcTRyQXJOalElMkZlSzZKcWgzQU5DOEJ0R3dDNFNXbnpJVjZOcnhtTmlmbTlDVGpYeXZEJTJCUTVRVGwyam5NNWxRNnh6aDBJT1NjRyUyQlZJZFdQYTB6THhGc0thSXZKM2tVdG1CRXg2aGJSJTJCWER5WVczT1BFcjNYVThVRVNzNTlINlJ2bHZJV2NCYlk4dzRTRjJaaEZGTjRydVZnZXBoaUdXckNGbWo3dTU0ekVXcW9NVUglMkJjU0s2SU5rSkdxeGNwYXlOM2xURFdBdVpONHZ3ZkdKUDFNMEZlTDdKUUkxY3c4OEdhdU5hMThrRUl0Vk51TTBoSUVQTmIxNVhXV3hBZ29nRnhYcWtEQjVLVjNRQVFvR1VBY0xTMjI2TElmWGlRTmMlMkZOZml4RFllNXVlaUt6eTVzanlSajJmb24xand3amFFJTJGRnpkQzZCSyUyQmRQUFlubUVsJTJGc1YxOHhwNEhtSHFRY1RSUGpJbGFxbzIwcSUyRm01ZDNucHp1JTJCSFhsbllCSmZiQTlhOTJPYlFsRFBoSFAxZnVNa2tYRkRvQnJnV09laEc4NHExS1NvWmhvTU5Rbk90eiUyRjUwRzVTc2czQ2JkJTJGUFRHb2RiZGZSbHU1SkxnOE5GTnVGQzl1eE9tcyUyQjZ2bjNjVVB0SWY0eU11bkhGSlNhVUVxNjhrV0diMWZvT1lqaXVEbDUxVVBKM2E4RUpmQ2g4JTJGVkJ1ZTZVSlg3MW1WUCUyQkxrOWpDS0s5UDJUcnJqclplaWtVJTJCQXFNZmoxdkJyQ1dXaHlMOW9YVzJnZCUyQkhjakdLRjdvclhCYjdVcEhvY2QxZThWY08xUmlZd1FZQ2xpNk5UZmowVm9sMlFoNGNPcHFPUkhVTW1uejJUdVVwcUZVMEI3MU1NM01MJTJGZWZoc1gwV3kxUlRqdDBhbjFLdUtKTEpsVTNQUGxyeUh2RUc4Q3VTZjRLTUdielMlMkZZbldkaWE1M1gwaEFUeURMVzRqaHNrYmIzNkpnZm9LbVc4M2klMkZIMHFQVkN3Wkl0RDlYTGYyQzZuN2VYczdSdjhTZHZXbWZKdnB3ck11YU93MEtNeUNncUoyNnAyJTJGTHBQNkJITHBBVG1jUzU4bVRZUU42dXd5V3g0NzBQaCUyQnJJRGpVcjJyWDhkNVBETmJmejVmOXNNZW8zcFpMM3h3bCUyRlg4TllKb2VGUnBkM1RVMnNjTThpQzE0bFpraThCJTJGcEpIc1c5T1pYU3VJTXBrRTg0MHh1WW9qdExqMWl5Vjh2NXU0SUV5RlBDZ0gzMW5yNzlZdmIyeFV4Y1hIY3FUdTZLQlZUbDlyeXkyUTFGYUl4JTJGNiUyQktQejZHb0M3bWclMkZITDZ6V1R3bEx4RURYJTJCczVFVE5rTXdKdnpjJTJGYWZURnQzMXh0ZjNESGZtT0dqbWRyc2JDVjloM2ZnJTJCUzB0WkJLOHRLT29GcjBhSmwlMkZIa2M3d2hyTm11WWN5QzRhbkJaNnZpdWE2QmdmWWVrVkR2clpETTEwbWF6ZTI5d3FZNkhvaU4lMkIyRHRyZkhKVElwTmRKUGdDbDZTekRGMmF1VHNlWXk5RUlQYTE2U3ZxTlFTaFAzcG11OGRvUXM4MlA5bUk1bkxQRTNDRCUyQkh2OGtSTEhBMzlmMlVNbTFHYTFzb0RXWW9JTmtyaVlQJTJGQ2pDdk1uRVVqOU1JbXRqczFIT0NxWlp6OUwlMkZLJTJCdXZLZnNZWUlySUR2VE1EeXk3bldjZzhONXQzMnRLaUo4dThvcHRwakQxVFg1UE5NcnFuWW5xdnhXZ0RjNDZWNWl0dmh2SjVIWFk2NyUyQmFTQnpWV3hxT3RWNlBjUHZUUFclMkZVRzVzT1h6emVmaG5sNWVSMWpuJTJCTXlVdmMzTEFrdXExdHVaVzdXTmIlMkJ6YXNHSDA2emY5eHdGSlc3aTVueEVoTzhPTFRNWWMxOWZFTk9lcXEyVEZpQTVJMHlhTlRWTlVOZmJGN2hNaG9ERmRaSmJuRWxPRTFibVlxcWFjVU9nJTJGcVVZVkwyZzdjN3kzdGxzeG0lMkZYYUdORFRQJTJCQm56VUh0M1FCcmNHVlJyQk11bDIxbmN5bkFpc1VVdHV4c08lMkZLVm4wNk1zWTRLbTYyTkIyT3A4d04lMkJSVVRldk1LeDJMTzFtQk4lMkJpS09WY3RiRGk4YjNXS082a2E1UWFxdWRBdHZ6cFJrYUFrbktzVjQ5VDVJUVdiTEhjMSUyRkFOWE93UW1HcmxWT0JhWEN2VFVndGphWm0lMkZvcFdPM2dxZmlMNDZTQW5ZajU3WDd2Y3dRTEc3YXJHSkIxeWF6WDZXRWEzT0w4RlpSSk4zbmpYcjhCRlczTyUyRndORVllbU4yUUhEWHZoY1FwSFJlVlhPSUlwcm81TDhPdGRZbWZlOERuVDRkJTJGSTh5U0hoSnN6dW5rQmQ0aG94RzIlMkZmcnYlMkZPcTNYOEdVam45cEttMlhzVHVWdnFmMWlpMktlN0FHZWxQWVQ3UW12SGhYT1RmMGNjVnhQN05rQyUyRiUyQkxjZ0kxTjJLdUs5QThmJTJGNnh5VlNSa1hRZXVLdnZrNTVrWmZ2N2Jva3NYWVdmdVBqT2tMNGM0bVhuNSUyRlJGb25HR05CQWl1Z3N6b0pqUkxLV1dBeTFwaVM4MWFkYWFsJTJGbEdVWDc5Y
```

Comment:
what is the difference between test accuracy and test_loss? need to write down in markdown. which loss is used?